src/StackExchange.Redis/PhysicalConnection.cs

```diff
@@ -248,6 +248,7 @@ private enum ReadMode : byte
         private readonly WeakReference _bridge;
         public PhysicalBridge? BridgeCouldBeNull => (PhysicalBridge?)_bridge.Target;
 
+        public long LastReadSecondsAgo => unchecked(Environment.TickCount - Thread.VolatileRead(ref lastReadTickCount)) / 1000;
```

Comment:
nit: changing this from seconds to LastReadMillisecondsAgo would eliminate some math

Reply:
Agreed, but since we have a sibling property for the Write side here, wanted this to match. Cheers on the math sanity checks and catching epic fail <3